Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tokio-mockstream"
-version = "1.1.0"
+version = "1.1.1"
```

Comment:
@lifehug because this is a fork of an otherwise public crate, I don't think this will work without collisions unless we change the name, as well.

Reply:
@dmcguire81 I wanted to follow up on this. I think we're okay with this as we have to attach the crate location. However, if it becomes an issue, we'll change the name.